x-pack/plugins/cloud_security_posture/public/application/app.tsx

```diff
@@ -51,15 +54,17 @@ export const CspApp = ({ core, deps, params }: CspAppDeps) => (
       <QueryClientProvider client={queryClient}>
         <EuiErrorBoundary>
           <Router history={params.history}>
-            <I18nProvider>
-              <Switch>
-                {routes.map((route) => (
-                  <Route key={route.path} {...route} />
-                ))}
-                <Route exact path="/" component={RedirectToDashboard} />
-                <Route path="*" component={UnknownRoute} />
-              </Switch>
-            </I18nProvider>
+            <CspThemeProvider>
```

Comment:
No need to wrap in a theme provider, [our entire app is wrapped in a `<KibanaThemeProvider />`](https://github.com/elastic/kibana/blob/0a4f0fc5046943564cba3ba9952ede180fa352ae/x-pack/plugins/cloud_security_posture/public/application/index.tsx#L21) which is a wrapper of `<EuiThemeProvider />`

Reply:
`KibanaThemeProvider` doesn't include the context for  `eui`'s emotion theme, which is what this bit adds

although if we manage to only use `useEuiTheme()` , we won't need this at all